build-support/bin/clang/linux/x86_64/5.0.1/build.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
```

Comment:
Do we actually need both of these? For my own sanity, it would be nice to publish the two versions independently/lazily if possible.

Reply:
These can be done entirely independently, the `build-clang.sh` script at the root directory (and the `build-clang-for-linux-with-cmake.sh` script) can be invoked with either version to produce the correct binary. I'm not sure what you're asking -- pantsbuild/pants#5490 defaults to version `6.0.0`, and the package for version `5.0.1` doesn't need to be updated, so I could just remove this script?

Reply:
Sure.